src/admin/usageStatus/deleteUsageStatusDialog.component.tsx

```diff
@@ -0,0 +1,95 @@
+import { AxiosError } from 'axios';
+import { ErrorParsing, UsageStatus } from '../../app.types';
+import React from 'react';
+import handleIMS_APIError from '../../handleIMS_APIError';
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Button,
+  Box,
+  FormHelperText,
+} from '@mui/material';
+import WarningIcon from '@mui/icons-material/Warning';
+import { useDeleteUsageStatus } from '../../api/usageStatus';
+
+export interface DeleteUsageStatusProps {
+  open: boolean;
+  onClose: () => void;
+  usageStatus: UsageStatus | undefined;
+}
+
+const DeleteUsageStatusDialog = (props: DeleteUsageStatusProps) => {
+  const { open, onClose, usageStatus } = props;
+
+  const [formError, setFormError] = React.useState<string | undefined>(
+    undefined
+  );
+
+  const { mutateAsync: deleteUsageStatus, isPending: isDeletePending } =
+    useDeleteUsageStatus();
+
+  const handleClose = React.useCallback(() => {
+    setFormError(undefined);
+    onClose();
+  }, [onClose]);
+
+  const handleDeleteUnit = React.useCallback(() => {
+    if (usageStatus) {
+      deleteUsageStatus(usageStatus.id)
+        .then(() => {
+          onClose();
+        })
+        .catch((error: AxiosError) => {
+          const response = error.response?.data as ErrorParsing;
+          if (response && error.response?.status === 409) {
+            setFormError(`${response.detail}. Please delete the Item first`);
```

Comment:
I don't think this is very user friendly as `The specified usage status is a part of an Item. Please delete the Item first`. Perhaps
```suggestion
            setFormError(`This usage status is currently used by one or more items. Remove all uses before deleting it here.`);
```